src/main/java/billgates/database/MySQLDatabaseGateway.java

```diff
@@ -506,7 +507,7 @@ public void modifyEntry(int billId, int entryId, String column, String newValue)
             // If equal, we are modifying the entry from the user's bill
             query = String.format("""
                         UPDATE bill_%d
-                        SET %s = "%s"
+                        SET `%s` = '%s'
```

Comment:
Does adding ticks (`) like this not affect how we insert the other types? 

Also, good catch with the double quotes " and single quotes '. I googled and it seems like single quote is more preferred.

Reply:
For keywords like `from`, if you do not quote them using ```, then you will get an error.